### PR TITLE
fix(sil_euro_latin): repeated ik, ff, gn, qu language codes

### PR DESCRIPTION
**Pull request by mcdurdin**
Relates to keymanapp/keyman#8118

### CODE REVIEW

**Comment by darcywong00** [comment]
I'd have warmer fuzzies if someone could load this .kmp on an iOS device/emulator :)

File: release/sil/sil_euro_latin/source/sil_euro_latin.kmn [hunk 1]
```diff
@@ -14,7 +14,7 @@ c KeymanWeb-specific header statements
 store(&KMW_HELPFILE) 'sil_euro_latin.html'
 store(&KMW_EMBEDJS) 'sil_euro_latin_js.txt'
 store(&LAYOUTFILE) 'sil_euro_latin.keyman-touch-layout'
-store(&KEYBOARDVERSION) '2.0.2'
+store(&KEYBOARDVERSION) '2.0.3'
```
Comment (darcywong00) [inline release/sil/sil_euro_latin/source/sil_euro_latin.kmn:17]: Should we also update l.149 in sil_euro_latin.kps?
```
     <Version>2.0.1</Version>
```

I think the fv_all build script uses the keyboard version defined in the .kps files

**Review by darcywong00 — APPROVED** [review]
lgtm

File: release/sil/sil_euro_latin/source/sil_euro_latin.kmn (hunk 1 above)
Comment (mcdurdin) [reply release/sil/sil_euro_latin/source/sil_euro_latin.kmn:17]: Problem here is that we use `<FollowKeyboardVersion/>` so Keyman Developer will remove the version number on save from the .kps. So a manual patch here is not going to work long-term. We may need to fixup fv_all/build.sh to read from the .kmp files.

**Comment by mcdurdin** [comment]
> I'd have warmer fuzzies if someone could load this .kmp on an iOS device/emulator :)

Done. 2.0.3 installs without issue.

Note: I've now also enabled build artifacts on pull request test builds.

**Pull request merged**